docs/data/material/getting-started/templates/checkout/PaymentForm.tsx

```diff
@@ -1,65 +1,272 @@
-            id="cvv"
-            label="CVV"
-            helperText="Last three digits on signature strip"
-            fullWidth
-            autoComplete="cc-csc"
-            variant="standard"
-          />
-        </Grid>
-        <Grid item xs={12}>
+    <Stack spacing={6} useFlexGap>
+      <FormControl component="fieldset" fullWidth>
+        <RadioGroup
+          aria-label="Payment options"
+          name="paymentType"
+          value={paymentType}
+          onChange={handlePaymentTypeChange}
+          sx={{
+            flexDirection: { sm: 'column', md: 'row' },
+            gap: 2,
+          }}
+        >
+          <Card
+            raised={paymentType === 'creditCard'}
+            sx={{
+              maxWidth: { sm: '100%', md: '50%' },
+              flexGrow: 1,
+              outline: '1px solid',
+              outlineColor:
+                paymentType === 'creditCard' ? 'primary.main' : 'divider',
+              backgroundColor:
+                paymentType === 'creditCard' ? 'background.default' : '',
+            }}
+          >
+            <CardActionArea onClick={() => setPaymentType('creditCard')}>
+              <CardContent sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+                <CreditCardRoundedIcon color="primary" fontSize="small" />
+                <Typography variant="subtitle1" fontWeight="medium">
+                  Card
+                </Typography>
+              </CardContent>
+            </CardActionArea>
+          </Card>
+          <Card
+            raised={paymentType === 'bankTransfer'}
+            sx={{
+              maxWidth: { sm: '100%', md: '50%' },
+              flexGrow: 1,
+              outline: '1px solid',
+              outlineColor:
+                paymentType === 'bankTransfer' ? 'primary.main' : 'divider',
+              backgroundColor:
+                paymentType === 'bankTransfer' ? 'background.default' : '',
+            }}
+          >
+            <CardActionArea onClick={() => setPaymentType('bankTransfer')}>
+              <CardContent sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+                <AccountBalanceRoundedIcon color="primary" fontSize="small" />
+                <Typography variant="subtitle1" fontWeight="medium">
+                  Bank account
+                </Typography>
+              </CardContent>
+            </CardActionArea>
+          </Card>
+        </RadioGroup>
+      </FormControl>
+      {paymentType === 'creditCard' && (
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'center',
+            alignItems: 'center',
+            gap: 2,
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              justifyContent: 'space-between',
+              p: 3,
+              height: { xs: 300, sm: 350 },
+              width: { xs: 450, sm: 525 },
```

Comment:
```suggestion
              width: "100%",
```

Ultimately, I think it harmonizes better if the card is spanning the full container width... That fixed size on mobile was also causing horizontal scroll